mitodl/open-discussions
# Remove sites app, add django.contrib.sites app

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #3587 

#### What's this PR do?
- Removes the current `sites` app and most of the associated frontend code
- Adds settings to replace `AuthenticatedSite` fields that are still used:
    - `title` -> `settings.OPEN_DISCUSSIONS_TITLE`
    - `tos_url` -> `settings.OPEN_DISCUSSIONS_TOS_URL`
    - `base_url` -> `settings.SITE_BASE_URL`
- Adds `django.contrib.sites` to `INSTALLED_APPS` with default `SITE_ID=1` in settings

#### How should this be manually tested?
- The site should still work the same as before.
- If you go to django admin, there should be one `Site` already and you should be able to add more
- If you change `OPEN_DISCUSSIONS_TITLE` and `OPEN_DISCUSSIONS_TOS_URL` from defaults in your .env file, that should be reflected in the top left hamburger menu "Terms and Conditions" link and the home page header title respectively.
- Tests should all pass


#### Any background context you want to provide?
Eventually a new site should be added for "InfiniteCorridor", and the models, views for field channels etc should associated with that site (future PR).
